src/tests.rs

```diff
@@ -1,11 +1,13 @@
 use html5ever::tree_builder::QuirksMode;
+use html5ever::QualName;
 use std::path::Path;
 
 use tempdir::TempDir;
 
-use crate::parser::parse_html;
-use crate::select::*;
-use crate::traits::*;
+use parser::{parse_html, parse_fragment};
+use select::*;
+use traits::*;
```

Comment:
These changes look like the cause of the CI failure, since they undo some 2018 syntax.